event-handler/teleport_events_watcher_test.go

```diff
@@ -136,6 +138,142 @@ func TestNext(t *testing.T) {
 	}
 }
 
+// errMockTeleportEventWatcher is Teleport client mock that returns an error after the first SearchUnstructuredEvents
+type errMockTeleportEventWatcher struct {
+	// events is the mock list of events
+	events []events.AuditEvent
+	t      *testing.T
```

Comment:
```suggestion
```

Doesn't appear that we need `t` at all here.

Reply:
we do, it's used in https://github.com/gravitational/teleport-plugins/pull/963/files#diff-814dbfbaa5bc225356b31467100e1ef90d9d707c13524f7471480f279ab64cbfR168